src/riak_cs_block_server.erl

```diff
@@ -311,18 +308,19 @@ handle_local_notfound(RcPid, FullBucket, FullKey, GetOptions2,
                       ProxyActive, ClusterID) ->
 
     Timeout = riak_cs_config:get_block_timeout(),
-    case get_block_local(RcPid, FullBucket, FullKey, GetOptions2, Timeout) of
+    case get_block_local(RcPid, FullBucket, FullKey, GetOptions2, Timeout,
+                         [riakc, get_block_n_all]) of
         {ok, _} = Success ->
             ProceedFun(Success);
 
         {error, Why} when Why == disconnected;
                           Why == timeout ->
-            _ = lager:debug("get_block_local/5 failed: {error, ~p}", [Why]),
+            _ = lager:debug("get_block_local() failed: {error, ~p}", [Why]),
```

Comment:
To be honest, I want to use standard or de-fact notation to refer a function that does not specify (or emphasize) its arity. Sometimes, `function_name/*` is used in Erlang, but it seems like a group of functions with the name (right?).
Any recommendation is appreciated :sweat_smile: 

Parenthesis with empty args may be not familiar.. Hmm... I saw the notation in exometer_core README.